src/pages/Proposal.vue

```diff
@@ -142,60 +143,44 @@ export default defineComponent({
 			ProposalType,
 		};
 	},
-	setup() {
-		const formState = ProposalStore();
-
-		const onFinish = (values: Event) => {
+	methods: {
+		onFinish(values: Event) {
 			console.log("Success:", values);
-		};
-		const onFinishFailed = (errorinfo: Event) => {
+		},
+		onFinishFailed(errorinfo: Event) {
 			console.warn("Failed:", errorinfo);
-		};
-		// eslint-disable-next-line
-		const disabledDate = (current: any) => {
+		},
+		disabledDate(current: any) {
 			// Can not select day before today
 			return (
 				current &&
 				current < new Date(new Date().valueOf() - 1000 * 60 * 60 * 24)
 			);
-		};
-
-		const range = (start: number, end: number) => {
+		},
+		range(start: number, end: number) {
 			const result = [];
 
 			for (let i = start; i < end; i++) {
```

Comment:
We can use range function at here, right?

Reply:
we don't have a direct range function in javascript, rather we use for loop

Reply:
Ah,  I see it.